providers/pool.rb

```diff
@@ -135,7 +135,11 @@ def configure
     doc = Document.new(xml)
 
     # root items
-    is_new_managed_runtime_version = new_value?(doc.root, 'APPPOOL/@RuntimeVersion', "v#{new_resource.runtime_version}")
+    if new_resource.no_managed_code
+      is_new_managed_runtime_version = new_value?(doc.root, 'APPPOOL/@RuntimeVersion', '')
+    else
+      is_new_managed_runtime_version = new_value?(doc.root, 'APPPOOL/@RuntimeVersion', "v#{new_resource.runtime_version}")
```

Comment:
@Matasx you added it here, it needs to be added to your git commit.

Run:
```
git commit --amend
```
edit the commit message to have `Signed-off-by: Your Name <your@email.com>`

then git push (to your github branches)